wallet_presentation/src/commonMain/kotlin/net/thechance/mena/wallet/presentation/screen/transaction_history/TransactionHistoryViewModel.kt

```diff
@@ -223,7 +224,7 @@ class TransactionHistoryViewModel(
     }
 
     private fun resetPaginator() {
-        updateState { it.copy(history = emptyList()) }
+        updateState { it.copy(history = emptyList(),) }
```

Comment:
```suggestion
        updateState { it.copy(history = emptyList()) }
```
i think the comma come when u format the file